uint32 flowerIndex
---
float32[] responseProbabilities
bool success